normandy/recipes/tests/test_filters.py

```diff
@@ -44,6 +45,18 @@ def test_uses_only_baseline_capabilities(self, settings):
             assert capabilities - settings.BASELINE_CAPABILITIES
 
 
+class TestProfileCreationDateFilter(FilterTestsBase):
+    def create_basic_filter(self, direction="olderThan", date="2020-02-01"):
+        return ProfileCreateDateFilter.create(direction=direction, date=date)
+
+    def test_generates_jexl(self):
+        filter = self.create_basic_filter()
+        assert set(filter.to_jexl().split("||")) == {
+            "(!normandy.telemetry.main)",
+            "(normandy.telemetry.main.environment.profile.creationDate<18262)",
```

Comment:
Since direction is "interesting", it would probably be good to have test for both values, and maybe some invalid values as well?

Reply:
Yes, right.